docs/README

```diff
@@ -0,0 +1,35 @@
+# How to build Sphinx documentation
+
+In order to build documentation for the standard library, first install sphinx
+
+```bash
+python -m venv build/
+source build/bin/activate
+pip install sphinx
+
+```
+
+Next, build gem5, then cd to the docs directory
+
+```bash
+scons build/ALL/gem5.opt
+cd docs
+```
+
+In order to generate the RST files, run `../build/ALL/gem5.opt gem5-sphinx-apidoc -o . ../src/python/gem5 -F -e`
```

Comment:
When I run this, I get the error `ModuleNotFoundError: No module named 'sphinx'`

Reply:
Let me double check this still works on my end, but did you run all the steps above to install sphinx? I think I've gotten that in the past if I forgot to start up the virtual environment